boards/arm/tdk_robokit1/board.cmake

```diff
@@ -1,4 +1,6 @@
 # SPDX-License-Identifier: Apache-2.0
 
+board_runner_args(jlink "--device=ATSAME70Q21A")
```

Comment:
There is no need for revision and you can drop it. Since boards can have both Rev A/B I would prefer drop it to avoid noise.

```diff
-ATSAME70Q21A
+ATSAME70Q21
```